rename_labels.json

```diff
@@ -59,7 +59,7 @@
         }
     ],
     "returns": {
-        "description": "The data cube with the renamed labels. The old labels can not be referenced any longer.",
+        "description": "The data cube with the same dimensions. The dimension properties (name, type, labels, reference system and resolution) have not changed, except that for the given dimension the labels have been changed. The old labels can not be referred to any longer. The number of labels has not changed.",
```

Comment:
> have not changed, except that for the given dimension the labels have been changed

remain the same, except that for the given dimension the labels change

> The number of labels has not changed.

The number of labels remains the same